FileUpload/src/FileUpload.jsx

```diff
@@ -67,7 +64,7 @@ export default class FileUpload extends Component {
         }
         if (nextProps.selectedFile.fileId !== this.state.selectedFile.fileId) {
             const file = nextProps.selectedFile;
-            this.updateFileState(file);
+            //this.updateFileState(file);
```

Comment:
Not needed anymore? Please consider removing

Reply:
I don't know who and why made this. This pull requests focus on project structure and build process.